RawWaterManagement/HydraulicSimulationResult/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,90 @@
+{
+  "id": "urn:ngsi-ld:dataModel:id:CWDU:80754394",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "1999-04-07T01:11:20Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2007-04-19T05:09:16Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "Moment knowledge where sea among role only. News prepare fact mother heavy. None music owner the."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Feeling pick almost century occur in understand. Somebody within page language."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "Suffer half effort happy couple. Friend return audience determine. Either part economy me career five network."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Much practice require occur."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Rather I traditional born. Center song choose agreement."
+  },
+  "owner": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:KXAQ:87468962",
+      "urn:ngsi-ld:dataModel:items:GJSJ:18395771"
+    ]
+  },
+  "seeAlso": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:AINS:90799655",
+      "urn:ngsi-ld:dataModel:items:YRIQ:09865111"
+    ]
+  },
+  "location": {
+    "type": "Property",
+    "value": {
+      "type": "Point",
+      "coordinates": [
+        -27.152203,
+        -138.726557
+      ]
+    }
+  },
+  "address": {
+    "type": "Property",
+    "value": {
+      "streetAddress": "Teach grow water cause also respond energy. Music specific save discuss contain wide rock. Her serve start hotel.",
+      "addressLocality": "Again fear finally particular form. Medical its maintain total. Phone eight culture space event bill tonight religious.",
+      "addressRegion": "Under would tend current. Assume ever tell speak. Specific information professor yes.",
+      "addressCountry": "Ago option shake piece. Way phone day cover law book just mean. Picture wait shoulder computer how dream.",
+      "postalCode": "Someone than successful garden challenge piece big. Rather history necessary authority. Make military shoulder itself nice quickly.",
+      "postOfficeBoxNumber": "Shake style determine name back doctor. Others account natural service. Key evening city outside keep.",
+      "areaServed": "Something nearly rest front. To night or themselves apply. Either base budget hundred arrive."
+    }
+  },
+  "areaServed": {
+    "type": "Property",
+    "value": "Take relationship audience little lay country arrive. Manager move move."
+  },
+  "type": "HydraulicSimulationResult",
+  "createdBy": {
+    "createdBy": "Version 0.1 not implemented",
+    "value": "urn:ngsi-ld:dataModel:createdBy:PNOG:25049917"
+  },
+  "hasInputOpenChannelSystem": {
```

Comment:
type should be Relationship
The proper way to model Relationships is as follow :

"observedBy":{
	"type":"Relationship",
	"object":" urn:ngsi-ld:Device:01 "	}